qiskit_dynamics/pulse/pulse_to_signals.py

```diff
@@ -254,3 +264,59 @@ def _get_channel(self, channel_name: str):
             raise QiskitError(
                 f"Invalid channel name {channel_name} given to {self.__class__.__name__}."
             ) from error
+
+
+def get_samples(pulse: SymbolicPulse):
```

Comment:
Add type hint for return type.

Reply:
added at https://github.com/Qiskit/qiskit-dynamics/pull/149/commits/4dab802b8e400282a022df72341da5aff353b5b0